test/summarize.test.js

```diff
@@ -0,0 +1,23 @@
+const fs = require('fs').promises;
+const summarize = require('../summarizer/summarize');
+
+
+describe('summarize tests...', () => {
+  let inputData;
+  let predicted;
+  let expected;
+
+  beforeAll(async () => {
+    inputData = await fs.readFile('./summarizer/input.txt', 'utf-8');
```

Comment:
Move these files into `test/`, and/or just inline the contents of the files into this test file so we don't even have to read from disk.